src/Exceptions/EntryNotFoundException.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace Statamic\Exceptions;
+
+class EntryNotFoundException extends \Exception
+{
+    protected string|int $id;
+
+    public function setEntry(string|int $id): object
```

Comment:
We can probably accept the entry ID as a parameter to the class's constructor, rather than needing a separate method.

Like what we do for the `FieldtypeNotFoundException`: https://github.com/statamic/cms/blob/4.x/src/Exceptions/FieldtypeNotFoundException.php#L17

Reply:
Haha. Erin said exactly the same thing. I used this approach as it's exactly how Laravel does it, so was sticking to the convention of the core framework. However, happy to use the parameter approach!

Reply:
Actioned.